spec/models/search/bookmark_search_form_spec.rb

```diff
@@ -244,6 +284,41 @@
         expect(result).to eq bookmark
       end
     end
+
+    context("when sorting") do
```

Comment:
There is a previous block that's structured like this:
```rb
describe "bookmarkable_search_results" do
  describe "sorting" do
  end
end
```
So for the bookmark sorting specs, can you put them under this structure?
```rb
describe "search_results" do
  describe "sorting" do
  end
end
```